include/swift/Basic/LangOptions.h

```diff
@@ -323,6 +323,9 @@ namespace swift {
     /// to the Swift language version.
     version::Version cxxInteropCompatVersion;
 
+    /// Import unannotated functions and types as unsafe.
+    bool SafeCXXInterop = false;
```

Comment:
Instead of introducing a new `bool` here, please add this as an `EXPERIMENTAL_FEATURE` to `Features.def`, alongside the existing ones for warning about unsafe code.

Reply:
Thanks! Indeed, this looks cleaner now.